File: prob_1/scala/main.scala


object HelloWorld {
  def main(args: Array[String]) {
    val testLimit = 1000
    var total = 0
    for( i <- 0 to (testLimit-1)){
      if(i % 3 == 0 || i % 5 == 0){
        total = total + i
      }
    }
    println(total + " should be 233168")
  }
}

File: prob_1/kotlin/main.kt
//kotlinr main.kt

fun main(args: Array<String>) {
  var testLimit = 1000
  var total = 0
  for(i in 1..testLimit-1){
    if(i % 3 == 0 || i % 5 == 0){
      total += i
    }
  }
  print(total)
  println(" should be 233168")
}
